### PR TITLE
docs: add small summaries to readme

### LINKED ISSUE #135: Short description in Installation overview
### Discussed in https://github.com/arcus-azure/arcus.backgroundjobs/discussions/106

<div type='discussions-op-text'>

<sup>Originally posted by **fgheysels** November 29, 2021</sup>
In our documentation, we have a page which lists all the components and shows the command that can be used to install a backgroundjob.

I wonder if it would be helpful if  that overview contains a short description (1 or 2 lines) that briefly describes what you can do with that job.  It might be even enough to just create a link to the extended documentation of that component ?</div>

### PR DESCRIPTION
**Pull request by stijnmoreels**
Adds small summaries to the available packages listed in the `README.md` file.

Closes #135

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,29 +10,25 @@ Background job development in a breeze.
 # Installation
 Easy to install it via NuGet:
```
Comment (fgheysels) [inline README.md:12]: This is confusing;
We have a first-level header 'installation', with directly underneath 3rd level headers.  

Then, we have a sentence which stops with a colon.   I would change that to 'The background jobs packages are easy to install via NuGet.'

Comment (fgheysels) [reply README.md:12]: I suppose we will have a similar 'introduction' text on the documentation website ?

Comment (stijnmoreels) [reply README.md:12]: On the docs site, we use the CloudEvents package as example and have the sentence: 'The Arcus BackgroundJobs can be installed via NuGet:'

Comment (stijnmoreels) [reply README.md:12]: Ok, I'll change the headers and the sentence with your proposal! 👍

**Review by fgheysels — APPROVED** [review]

**Pull request merged**